### PR TITLE
added Rise filter

### PR DESCRIPTION
**Pull request by shelune**
Here's my try at the Rise filter. First time contributing to a GitHub project :')

### CODE REVIEW

**Comment by una** [comment]
Try to use the same convention as the other filters. Check out any of the files inside of *scss/* as an example. I.e: https://github.com/una/CSSgram/blob/master/source/scss/earlybird.scss

Also, test it on the test page! It would be at localhost:8000/test :) thanks!

**Comment by shelune** [comment]
added files, hope this will suffice

File: source/scss/rise.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Rise
+ *
+ */
+@import 'shared';
+
+%_rise,
```
Comment (una) [inline source/scss/rise.scss]: I would remove the leading underscore in the class name

File: source/scss/rise.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Rise
+ *
+ */
+@import 'shared';
+
+%rise,
+.rise {
+  @extend %filter-base;
+  filter: brightness(1.2) contrast(.9) saturate(.9);
```
Comment (una) [inline source/scss/rise.scss]: try: `brightness(1.05) sepia(.2) contrast(.9) saturate(.9)`

File: source/scss/rise.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  &:after{
+    background: radial-gradient(circle, rgba(232, 197, 152, .8), transparent 90%);
+    mix-blend-mode: overlay;
+    opacity: .6;
+  }
+
+  &:before{
+      background: radial-gradient(circle, rgba(236, 205, 169, .2) 55%, rgb(50, 30, 7) 90%);
```
Comment (una) [inline source/scss/rise.scss]: the outer vignette is really dark -- i'd try something more like `radial-gradient(circle, rgba(236, 205, 169, .15) 55%, rgba(50, 30, 7, .4));` and remove the opacity

File: site/test/index.html [hunk 4]
```diff
@@ -2109,6 +2109,97 @@ <h2 class="filter-header"><a href="#xpro2">Xpro2</a> <span class="done  done--ye
...
+
+      <div class="headers">
+        <h3>Instagram</h3>
+        <h3>CSSgram</h3>
+      </div>
+
+              <section class="test-images">
+          <div>
+            <figure>
+              <img src="ref/atx-rise.jpg" alt="Instagram version of rise">
```
Comment (una) [inline site/test/index.html]: thanks for adding these!

**Pull request closed without merging**